docs/udf.rst

```diff
@@ -332,11 +340,16 @@ built-in functions. The function names are case-insensitive. For example, using
 
 
 
-============
+=================
+UDFs and Security
```

Comment:
Since you renamed to "KSQL Custom Functions" (as the superset of UDFs and UDAFs), let's rename this to "KSQL Custom Functions and Security"?